jobs/periodic/anonymise-attendance-log/index.ts

```diff
@@ -0,0 +1,18 @@
+import { getLogger } from "log4js";
+import { getManager } from "typeorm";
+import { CourseAttendanceLog } from "../../../common/entity/CourseAttendanceLog";
+
+const DELETE_AFTER = 7; // in days
+const logger = getLogger();
+
+export default async function execute() {
+    const entityManager = getManager();
+    await entityManager
```

Comment:
Prisma please :) 
And please also check that this does not also delete the pupil, the schema has a 'cascading' setting and I'm not sure what that does.